src/weapon/w_050.c

```diff
@@ -192,9 +192,207 @@ void func_162000_8017BA38(Entity* ent, Point16* outPoint, bool arg2) {
 
 INCLUDE_ASM("weapon/nonmatchings/w_050", func_ptr_80170004);
 
-INCLUDE_ASM("weapon/nonmatchings/w_050", func_ptr_80170008);
+void func_ptr_80170008(Entity* self) {
+    const int PrimCount = 16;
```

Comment:
Nice idea!

Reply:
I think you suggested it in https://github.com/Xeeynamo/sotn-decomp/pull/1401#discussion_r1673048922 — so yeah, good idea! 😄